### DISCUSSION TITLE
Error while Running Inteferogram

### DISCUSSION THREAD

**Discussion by galuhqfs** [gmtsar/gmtsar#362] [Q&A]
Hi, I have an issue while running interferogram using SBAS method. Nothing wrong while processing until I run a command :
`intf_tops.csh intf.in batch_tops.config`. 

I already edited the batch_tops.config based on the tutorial. The result is :

If: Badly formed number.

I have no idea what's wrong with the process cause everything seems normal for me. I am a beginner for using GMTSAR. I really hope this issue can be solved. Thanks

**Comment by Xiaohua-Eric-Xu** [comment]
`If: Badly formed number.` feels like some strange characters getting into your intf.in. Did you edit it after it is produced by select_pairs.csh?

**Reply by galuhqfs** [reply]
Hi, Eric. Thank you for the respond.

Accidently, I didn't do anything with the intf.in. Fortunately, my problems is already solved by converting the `batch_tops.config` using `dos2unix`.

Thank you.

**Comment by chintals** [comment]
Can you check for an extra newline character in the intf.in file (possibly after the last intf pair) and remove it.

**Reply by galuhqfs** [reply]
Hi, Chintals. Thank you for responding my problem. Nothing wrong with my intf.in file. But fortunately, my problems is already solved by converting the `batch_tops.config` using `dos2unix`.
